drivers/unix/os_unix.cpp

```diff
@@ -728,6 +728,11 @@ bool OS_Unix::is_process_running(const ProcessID &p_pid) const {
 	return true;
 }
 
+int OS_Unix::get_process_exit_code(const ProcessID &p_pid) const {
+	// TODO
+	return -1;
```

Comment:
Should be something like:

```suggestion
	int status = 0;
	if (waitpid(p_pid, &status, WNOHANG) != 0) {
		return = WIFEXITED(status) ? WEXITSTATUS(status) : status;
	}
	return -1;
```